src/index.html

```diff
@@ -0,0 +1,177 @@
+            <ul class="block__list">
+              <li class="block__item">Information architecture</li>
+              <li class="block__item">Interface design</li>
+              <li class="block__item">Product Design</li>
+              <li class="block__item">Integrated ad Companies</li>
+            </ul>
+          </div>
+        </div>
+        <div class="choose__view">
+          <img src="./assets/images/chooseimage.png" alt="choose image" class="choose__image">
+        </div>
+      </div>
+    </section>
+    <section class="portfolio wrapper__portfolio">
+      <div class="info portfolio__info">
+        <h3 class="info__title info__title_center info__title_left">our portfolio</h3>
+        <p class="info__subtitle info__subtitle_ta-center info__subtitle_w-765 info__subtitle_left">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+      </div>
+      <div class="portfolio__body">
+        <div class="categories portfolio__categories">
+          <h3 class="categories__title">choose category</h3>
+          <ul class="categories__list">
+           <li class="categories__item categories__item_active"><a href="#!" class="categories__link categories__link_active">all</a></li>
+           <li class="categories__item"><a href="#!" class="categories__link">webdesign</a></li>
+           <li class="categories__item"><a href="#!" class="categories__link">graphic design</a></li>
+           <li class="categories__item"><a href="#!" class="categories__link">fashion</a></li>
+           <li class="categories__item"><a href="#!" class="categories__link">logo design</a></li>
+           <li class="categories__item"><a href="#!" class="categories__link">advertising</a></li>
+        </ul>
+      </div>
+      <div class="gallery portfolio__gallery">
+       <div class="gallery__column">
+          <img src="./assets/images/stairs.png" alt="stairs" class="gallery__img">
+          <img src="./assets/images/buttles.png" alt="buttles" class="gallery__img">
+          <img src="./assets/images/beer.png" alt="beer" class="gallery__img">
+        </div>
+        <div class="gallery__column gallery__column_hide2">
+          <img src="./assets/images/forest.png" alt="forest" class="gallery__img">
+          <img src="./assets/images/paper.png" alt="paper" class="gallery__img">
+          <img src="./assets/images/milk.png" alt="milk" class="gallery__img">
+        </div>
+        <div class="gallery__column gallery__column_hide3">
+          <img src="./assets/images/box.png" alt="box" class="gallery__img">
+          <img src="./assets/images/water.png" alt="water" class="gallery__img">
+          <img src="./assets/images/pencils.png" alt="pencels" class="gallery__img">
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="subscribe wrapper__subscribe">
+    <div class="subscribe__body">
+      <div class="subscribe__leftSide">
```

Comment:
If you use BEM for name class you nid use `-` for separate two words. for example `subscribe__left-side`

Reply:
Done.